test/assert_relationship_test.exs

```diff
@@ -99,4 +99,9 @@ defmodule AssertRelationshipTest do
         assert "record with `id` 1 and `type` \"post\" was found but had mis-matching attributes" == error.message
     end
   end
+
+  test "will return the original payload" do
+    payload = assert_relationship(data(:payload), data(:author), as: "author", for: data(:post))
```

Comment:
That being said, it would be nice if the `assert` macro could return the result of whatever is being run. This way you could use it to create composeable assertions